assets/js/script.js

```diff
@@ -62,4 +61,37 @@ var bookFetchHandler = function (searchTerm) {
         });
 };
 
+var bookObjectCreator = function (data) {
+    // use a random number to select index of returned data to ensure
+    // new books are discovered upon each search
+    var randomNumber = Math.floor(Math.random() * data.items.length);
+    // get title information
+    var title = data.items[randomNumber].volumeInfo.title;
+    // get image url
+    var imageUrl = data.items[randomNumber].volumeInfo.imageLinks.thumbnail;
+    // get description
+    var description = data.items[randomNumber].volumeInfo.description;
+    if (!description) {
+        description = "Description is unavailable for this content.";
+    };
+    // define "authors" location in data
+    var authorsArray = data.items[randomNumber].volumeInfo.authors;
+    // save all authors in array
+    var authors = [];
+    for (i = 0; i<authorsArray.length; i++) {
+        var newAuthor = authorsArray[i];
+        authors.push(newAuthor);
+    }
```

Comment:
The authors are already stored in an array in the response. No need to make a new one.
console.log(authors) === console.log(authorsArray)

Reply:
Yeah, I see what you mean. I did that because it was only sending the first index of the authors array to the author property in the object, but I could obviously just loop through the original and push to the object each time. Thanks Zachary, it’s really nice to have another pair of eyes to point these things out. I’ll get on that next time I’m at the computer.